clients/src/main/resources/common/message/DescribeLogDirsRequest.json

```diff
@@ -19,7 +19,7 @@
   "listeners": ["zkBroker", "broker"],
   "name": "DescribeLogDirsRequest",
   // Version 1 is the same as version 0.
-  "validVersions": "0-2",
+  "validVersions": "0-3",
```

Comment:
nit: Could we also put a comment for version 3?

Reply:
Yes, good idea. Done!